Create new Architecture and Styling blocks (#11)
WeDevelop-ARG/full-stack-beginner-program: content/block-2/index.md

### React

- Tutorial by React: https://reactjs.org/tutorial/tutorial.html (use the *Local environment setup*).
- React Hooks: https://reactjs.org/docs/hooks-intro.html

> ### Github ??? (maybe a better title like: Collaboration)

Reviewer: ```suggestion
### Git Flow in practice
```

> - TODO: How to create a Pull Request: https://

Reviewer: ```suggestion
- How to create a Pull Request: https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request
```

- How to Write a Git Commit Message: https://chris.beams.io/posts/git-commit/

## Tasks

1. Create a public repository into your Github account to host the game example from the tutorial. Make sure to put in practice the [Git Flow](https://datasift.github.io/gitflow/IntroducingGitFlow.html).

> 2. Create a new Pull Request extracting components into its own files (TODO: FE Arch ???) and converting React Class Components into Functional Components.

Reviewer: FE architecture is not the focus of this block. They'll be doing that in B4.
```suggestion
2. Create a new Pull Request extracting components into its own files and converting React Class Components into Functional Components.
```
